src/modal-v2/slots/card.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { isStyledComponent } from "styled-components";
+import { ModalCardProps } from "../types";
+import { CloseButton } from "./close-button";
+import { Content } from "./content";
+import { ModalCard } from "./slot-styles";
+
+export const Card = ({
+    id,
+    "data-testid": testId = "modal-card",
+    children,
+    ...otherProps
+}: ModalCardProps) => {
+    // =============================================================================
+    // EVENT HANDLERS
+    // =============================================================================
+    const handleOnClick = (event: React.MouseEvent) => {
+        event.stopPropagation();
+    };
+
+    const findByType = (child: React.ReactPortal, type: any) =>
```

Comment:
1. put this under a `HELPERS` section
2. rename to boolean-y function

```suggestion
    const isComponentType = (child: React.ReactPortal, type: any) =>
```